src/components/Router/index.js

```diff
@@ -74,7 +76,8 @@ class MainRouter extends Component {
   mobileConfig = () => {
     const {documentType, options} = this.props
     const {steps, token} = options
-    return {steps, token, documentType, step: this.state.mobileInitialStep}
+    const woopraCookie = window.onfidojssdkwoopra.cookie
```

Comment:
this reference to window can still be removed, by having a `getWoopraCookie` function.

Reply:
fixed